test/Sanitizers/tsan/racy_async_let_fibonacci.swift

```diff
@@ -36,6 +27,7 @@ func fib(_ n: Int) -> Int {
 
 var racyCounter = 0
 
+@available(SwiftStdlib 5.5, *)
```

Comment:
```suggestion
```

Could be removed since the `-disable-availability-checking` was passed.

Reply:
These tests mirror what is in `test/Concurrency/Runtime/async_let_fibonacci.swift`. If this is worth cleaning up, please apply the changes consistently across all versions.